### PR TITLE
feat/chrome-extension-hello-world

### PR DESCRIPTION
**Pull request by g4titanx**
completing task:
- [ ]  Add a hello world chrome extension with docs on how to set up dev env to use it in CONTRIBUTING.md (200 AI16z)

sol add: BjQWDZuX8o4BGi2dip9sgR5CnQ6XwCXoYPNFTSHxfdBT

image

### CODE REVIEW

**Comment by g4titanx** [comment]
@roninjin10

**Comment by roninjin10** [comment]
Can you rebase? Make sure to remove all the stuff that git ignore should be removing like node modules

**Comment by g4titanx** [comment]
> Can you rebase? Make sure to remove all the stuff that git ignore should be removing like node modules

done @roninjin10

File: app/extension/tsconfig.json [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
+    "target": "ES2020",
+    "module": "ES2020",
+    "moduleResolution": "node",
+    "esModuleInterop": true,
+    "outDir": "dist",
+    "rootDir": "src",
+    "strict": true,
+    "lib": ["ES2020", "DOM"]
+  },
+  "include": ["src"]
```
Comment (roninjin10) [inline app/extension/tsconfig.json]: Let's keep the stricter old settings

Comment (roninjin10) [reply app/extension/tsconfig.json]: We will need to install that @tevm/tsconfig package

File: app/extension/package.json [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
+    "watch": "tsc --watch"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "MIT",
+  "dependencies": {
+    "@onchain-clippy/light-client": "workspace:^"
+  },
+  "devDependencies": {
+    "@types/chrome": "^0.0.260",
```
Comment (roninjin10) [inline app/extension/package.json:20]: will need to add @tevm/tsconfig as dep here

**Review by roninjin10 — CHANGES_REQUESTED** [review]
Small fixes from previous prs

**Comment by g4titanx** [comment]
done @roninjin10

**Review by roninjin10 — APPROVED** [review]

**Comment by roninjin10** [comment]
Thank you will send Christmas gift

**Pull request merged**